src/multicellular_stage/editor/CellBodyPlanEditorComponent.GUI.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using Godot;
+using Systems;
+
+/// <summary>
+///   The partial class containing GUI updating actions
+/// </summary>
+public partial class CellBodyPlanEditorComponent
+{
+    protected override void OnTranslationsChanged()
+    {
+        organismStatisticsPanel.OnTranslationsChanged();
```

Comment:
Doesn't the cell editor also need this method? I checked in my IDE and it sees only one call to `organismStatisticsPanel.OnTranslationsChanged()` which is from here so that would mean the microbe editor is missing this part...